src/components/LeftSidebar/ConversationsList/Conversation.vue

```diff
@@ -182,7 +182,7 @@ export default {
 
 	setup(props) {
 		const { item, isSearchResult } = toRefs(props)
-		const { counterType, conversationInformation } = useConversationInfo({ item, isSearchResult })
+		const { counterType, conversationInformation } = useConversationInfo({ item, isSearchResult, exposeMessages: true })
```

Comment:
Maybe in LeftSidebar as well?

![Screenshot from 2024-04-29 15-44-36](https://github.com/nextcloud/spreed/assets/93392545/8913bd17-ca94-4880-8269-76d8fa3ea298)

Reply:
fair enough

Reply:
We may need to find a styling differentiation between message and description at some point.

Reply:
Message contains an author's name, which could be highlighted. But something for later, yes